### PR TITLE
should dispatch pointer evens on touch events in Chrome(closes #3948 in TestCafe)

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

File: src/client/sandbox/event/simulator.ts [hunk 1]
```diff
@@ -468,16 +472,54 @@ export default class EventSimulator {
         return null;
     }
 
+    _getPointerEventTypeInfo (type: string) : { eventType: string, pointerType: string } {
+        const mouseToPointerEventTypeMap = {
```
Comment (miherlosev) [inline src/client/sandbox/event/simulator.ts]: Move `mouseToPointerEventTypeMap` and `touchToPointerEventTypeMap` constans on the module level.

File: src/client/sandbox/event/simulator.ts [hunk 2]
```diff
@@ -487,16 +529,16 @@ export default class EventSimulator {
             tiltY:       0,
             // NOTE: This parameter must be "1" for “mouse”.
             pointerId:   1,
-            pointerType: 'mouse',
+            pointerType: pointerType,
```
Comment (miherlosev) [inline src/client/sandbox/event/simulator.ts]: Can be simplifier as `pointerType,`

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

File: src/client/sandbox/event/simulator.ts [hunk 3]
```diff
@@ -32,6 +32,21 @@ const FOCUS_IN_OUT_EVENT_NAME_RE = /^focus(in|out)$/;
 // NOTE: initTextEvent method required INPUT_METHOD param in IE
 const DOM_INPUT_METHOD_KEYBOARD = 1;
 
+const mouseToPointerEventTypeMap = {
```
Comment (miherlosev) [inline src/client/sandbox/event/simulator.ts]: Rename them in upper case notation.
`MOUSE_TO_POINTER_EVENT_TYPE_MAP`, `TOUCH_TO_POINTER_EVENT_TYPE_MAP`

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Pull request merged**